{{cookiecutter.github_repo}}/requirements/production.txt

```diff
@@ -2,4 +2,4 @@
 -r common.txt
 
 # Import project package from repository
-{{cookiecutter.python_package_namespace}}
+-e ./{{cookiecutter.python_package_namespace}}
```

Comment:
@MikeTheCanuck This is the line that `pip install`s the python package from the local filesystem as opposed to the remote PyPI site.